### PR TITLE
first cut of a sample json file for review

### PR DESCRIPTION
**Pull request by ghost**
Here's a rough JSON file that follows the entity model. I'm curious to see if it supports what we have been doing for design work. I tried to make it in such a way that it could be replaced with a REST API.

### CODE REVIEW

File: src/service/example.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            {
+                "id": 111,
+                "name": "openshift-launchpad",
+                "abbreviation": "OSLAUNCH",
+                "shortDescription": "A bare bones starter kit that anyone can understand and use to build a 3-tier app and deploy to OpenShift",
+                "longDescription": "This component was created to meet the perceived demand for something that would improve the initial experience of new teams working in the BCDevExchange environment and on OpenShift in particular. The amount of work that a new (unfamiliar) team needs to do to get something simple running is consistently about 3 weeks. Using this product as a boilerplate, they can have 'hello world' running and deployed in OpenShift much faster and then begin creating code that provides business value sooner.",
+                "gitHubLink": "https://github.com/bcgov/openshift-launchpad",
+                "supportAgreement": null,
+                "sampleImplementationLink": null,
+                "ownerId": 333,
```
Comment (hitankar-freshworks) [inline src/service/example.json]: For this iteration, I was thinking that ownerId should be an owner object that links to the team.
```suggestion
                "owner": {
                        "id": 333,
                        "teamName": "Team CoCo",
                        "contactEmail": "rachit@freshworks.io",
                        "location": "CSI Lab 3rd floor",
                        "organizationalUnit": "Ministry of Citizen Services, BCDevExchange"
                }
```

**Review by hitankar-freshworks — APPROVED** [review]
We can restructure later

File: src/service/example.json (hunk 1 above)
Comment (ghost) [reply src/service/example.json]: I don't get it. You want to inline the team? The team is used in 3 places: product owner, coco owner, testimonial provider. I'm not sure copying and pasting them all over the place is a good move. A team will own more than one coco or product. I get keeping it simple, but it's pretty simple already.
Would you suggest we do the same thing for the team that owns the product and the team that provides the testimonial?

Comment (hitankar-freshworks) [reply src/service/example.json]: I am comparing this to what I'd expect for an API endpoint. I was a bit confused, but rechecking it seems you have multiple top-level objects. 

I guess you envision we will be getting owner information by searching the id in teams object. right?

**Review by hitankar-freshworks — APPROVED** [review]

**Review by ashtonmeuser — APPROVED** [review]

**Pull request merged**